recipes/napari-DeepSpot/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "napari-DeepSpot" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-DeepSpot-{{ version }}.tar.gz
+  sha256: 963c550cdea4da2bbb3b2bc865d1c3f66baf826d43afd5272ba2470febb8ebe0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - opencv
+    - pyqt
```

Comment:
This might cause problems in the bundle, since it will be distributed with `pyside`... I guess we need dev guidelines to use `qtpy` or similar wrappers.

Reply:
Actually I think we can just remove it, right? Whatever provider will be installed by `napari`. As long as the plugin does not import directly from `pyqt`...

```suggestion
```